packages/docs/src/routes/docs/concepts/resumable/index.mdx

```diff
@@ -27,13 +29,15 @@ Collectively, this is known as hydration. All current generations of frameworks
 
 Qwik is different because it does not require hydration to resume an application on the client. Not requiring hydration is what makes the Qwik application startup instantaneous.
 
-All other frameworks' hydration **replays** all the application logic in the client. Qwik instead pauses execution in the server, and resumes execution in the client.
+All other frameworks' hydration **replays** all the application logic in the client. Qwik instead pauses execution in the server , and resumes execution in the client.
+[//]: # (The very consept of pausing the application is huge, a schema or a quick explaination could help making a mental model.)
 
 ## Introducing Resumability
 
 Resumability is about pausing execution in the server and resuming execution in the client without having to replay and download all of the application logic.
 
 A good mental model is that Qwik applications at any point in their lifecycle can be serialized and moved to a different VM instance (server to browser). There, the application simply resumes where the serialization stopped. No hydration is required. This is why we say that Qwik applications don't hydrate; they resume.
+[//]: # (This is confusing, why speak about another VM, which makes think about some server instance, when right after it says (server to browser). Can we not simply say that apps are anyway serialized and sent to the browser ? Also why would serialization stop ?)
```

Comment:
Yes... it should come as a metaphor at the beginning, saying the "pausing and resuming are just like in VMs, where you pause the VM and send it to another computer.. with Qwik you pause the application on the server and send it inside the smart HTML file to the client, where it resumes"